### PR TITLE
[WINC-921]: Follow-up add fast-forward workflow

### PR DESCRIPTION
**Pull request by alinaryan**
Removes the 'git log' line and sets the upstream branch as origin in the case of fast-forwarding to a new branch.

Follow-up to #35781

### CODE REVIEW

File: ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh [hunk 1]
```diff
@@ -61,9 +61,8 @@ if ! git clone -b "$DESTINATION_BRANCH" "$repo_url" ; then
         exit 1
     fi
 
-    echo "INFO Pushing the following commits to $DESTINATION_BRANCH to origin"
-    git --no-pager log --pretty=oneline origin/"$DESTINATION_BRANCH"..HEAD
-    if ! git push origin $DESTINATION_BRANCH; then
+    echo "INFO Pushing to new branch $DESTINATION_BRANCH"
+    if ! git push --set-upstream origin "$DESTINATION_BRANCH" ; then
```
Comment (aravindhp) [inline ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh:65]: Please mention why you are making this change in your commit message.

I also don't understand why you need the `--set-upstream`. Why is that needed given it is an ephemeral config change?

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for the follow up, @alinaryan

File: ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh (hunk 1 above)
Comment (alinaryan) [reply ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh]: Updated the commit message and removed the `--set-upstream` given what you said. I was able to run this locally with no errors.

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Comment by saifshaikh48** [comment]
/remove-lgtm

for [`tools` instead of `cli`](https://github.com/alinaryan/release/blob/master/ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-ref.yaml#L3)

**Review by aravindhp — APPROVED** [review]
Thanks for fixing this up, @alinaryan 

/hold

for other reviews.

**Comment by mtnbikenc** [comment]
LGTM
